README.md

```diff
@@ -52,13 +53,18 @@ script:
 - BEFORE_SCRIPT: (default: not set): Used to specify shell commands or scripts that run before building packages.
 - UPSTREAM_WORKSPACE (default: debian): When set as "file", the dependended packages that need to be built from source are downloaded based on a .rosinstall file in your repository. When set to a "http" URL, this downloads the rosinstall configuration from an http location
 - TEST_BLACKLIST: Allow certain tests to be skipped if necessary (not recommended)
+- TEST: allow other tests to be run, such as code format checking using clang-format
 
-More configurations as seen in [industrial_ci](https://github.com/ros-industrial/industrial_ci) can be added, in the future.
+More configurations as seen in [industrial_ci](https://github.com/ros-industrial/industrial_ci) can be added in the future.
 
 ## Removed Configuration
 
 - ROS_REPOSITORY\_PATH: (UNSUPPORTED) replaced by ROS\_REPO
 
+## Clang-Format
+
+A new test is available that checks if the code is properly formatted as specified in the clang-format file found [here](https://github.com/davetcoleman/roscpp_code_format). Use ``TEST=clang-format`` to enable this test.
```

Comment:
I think the format definition file should be part of the MoveIt repos too, probably moveit_ci.